stable/datadog/README.md

```diff
@@ -12,17 +12,35 @@ Kubernetes 1.4+ or OpenShift 3.4+ (1.3 support is currently partial, full suppor
 
 ## Installing the Chart
 
-To install the chart with the release name `my-release`, retrieve your DataDog API key from your [Agent Installation Instructions](https://app.datadoghq.com/account/settings#agent/kubernetes) and run:
+To install the chart with the release name `my-release`, retrieve your Datadog API key from your [Agent Installation Instructions](https://app.datadoghq.com/account/settings#agent/kubernetes) and run:
 
 ```bash
 $ helm install --name my-release \
     --set datadog.apiKey=YOUR-KEY-HERE stable/datadog
 ```
 
-After a few minutes, you should see hosts and metrics being reported in DataDog.
+After a few minutes, you should see hosts and metrics being reported in Datadog.
 
 **Tip**: List all releases using `helm list`
 
+### Enabling the Datadog Cluster Agent ###
+
+Read about the Datadog Cluster Agent in the [official documentation](https://docs.datadoghq.com/agent/kubernetes/cluster/).
+
+Run the following if you want to deploy the chart with the Datadog Cluster Agent.
+You can also specify `clusterAgent.metricsProvider.enabled=true` if you want to enable the External Metrics Server.
+If you want to learn to use this feature, you can check out this [walkthrough](https://github.com/DataDog/datadog-agent/blob/master/docs/cluster-agent/CUSTOM_METRICS_SERVER.md).
+```bash
+helm install --name datadog-monitoring \
+    --set clusterAgent.token=YOUR-32-CHARACTERS-TOKEN \
```

Comment:
actually i think we should just get rid of this setting and generate a random token ourselves with `{{ randAlphaNum 32 | quote }}`. WDYT?

Reply:
smart